src/compass/utils/defaults/geo_cslc_s1.yaml

```diff
@@ -10,6 +10,18 @@ runconfig:
           safe_file_path:
           # Required. List of orbit (EOF) files (min=1)
           orbit_file_path:
+          # Required. Path to annotation file
+          annotation_file_path:
+          # Required. Path to RFI file
+          rfi_file_path:
+          # Required. Path to noise file
+          noise_file_path:
+          # Required. Path to calibration file
+          calibration_file_path:
```

Comment:
@vbrancat as we discussed offline, I think the easiest approach would be to not expose any of these four files for IF delivery. They all exist in the  safe_file_path for now.